docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,223 @@ across groups and also the difference and ratio between the maximum and minimum:
+    hires carelessly selected applicants at the same rate p. Even though 
+    the acceptance rates in both groups are identical, it is far more likely 
+    that unqualified applicants are selected in one group than in the other. 
+    As a result, it will appear in hindsight that members of group b 
+    performed worse than members of group a, thus establishing a negative 
+    track record for group b."
+
+It's also worth considering whether the assumptions underlying demographic
+parity maintain construct validity (see :ref:`construct_validity`). 
+Construct validity is a concept in the social sciences that assess the 
+extent to which the ways we choose to measure abstract 
+phenomena is valid. For demographic parity, one relevant question would be 
+whether demographic parity meets the criteria for establishing "fairness", 
+itself an unobservable theoretical construct. Further, it's important 
+to ask whether satisfying demographic parity actually brings us closer 
+to the world we'd like to see. 
+
+
+.. _conditional_group_fairness:
+
+In some cases, we may observe a trend in several groups of data, but that 
+trend may disappear or reverse when groups are combined. Known as  
+`Simpson's Paradox <https://en.wikipedia.org/wiki/Simpson%27s_paradox>`_, this 
+outcome may appear when observing disparate outcomes across groups. A 
+famous example of Simpson's Paradox is a study of 1973 graduate school 
+admissions to the University of California, Berkley :footcite:`bickel1975biasinadmissions`. 
+The study showed that when observing admissions by gender, men applying were 
+more likely than women to be accepted. However, drilling down into admissions 
+by department revealed that women tended to apply to departments with more 
+competitive admissions requirements, whereas men tended to apply to less 
+competitive depoartments. The more granular analysis showed only four out of 
+85 departments exhibited bias against women, and six departments exhibited 
+bias towards men. In general, the data indicated departments exhibited a bias 
+in favor of minority-gendered applicants, which is opposite from the trend 
+observed in the aggregate data.
+
+This phenomenon is important to fairness evaluation because metrics like 
+demographic parity may be different when calculated at an aggregate level and  
+within more granular categories. In the case of demographic parity, we might 
+need to review 
+:math:`\E[h(X) \given A=a, D=d] = \E[h(X) \given D=d] \quad \forall a` 
+where D represents the feature(s) within X across which members of the groups 
+within A are distributed. 
+Demographic parity would then require that the prediction of the target  
+variable is statistically independent of sensitive attributes conditional 
+on D. Simply aggregating outcomes across high-level categories can be 
+misleading when the data can be further disaggregated. 
+More granular categories generally contain smaller sample sizes, and 
```

Comment:
Could we move this sentence ("More granular categories...") down to the start of the next paragraph (maybe starting it with a "However" to indicate it's about issues)? So that whole next paragraph would then be about statistical limitations or potential threats to validity - e.g., small sample sizes and multiple comparisons.

Does that work for you?

Reply:
Done!